go-controller/pkg/clustermanager/node/node_allocator.go

```diff
@@ -413,8 +438,8 @@ func (na *NodeAllocator) Cleanup() error {
 		}
 
 		hostSubnetsMap := map[string][]*net.IPNet{networkName: nil}
-		// passing util.InvalidNetworkID deletes the network id annotation for the network.
-		err = na.updateNodeNetworkAnnotationsWithRetry(node.Name, hostSubnetsMap, util.InvalidNetworkID, nil)
```

Comment:
oopsy this should have been in commit3 but got sneaked into here..not a biggie though